.changeset/wise-donuts-check.md

```diff
@@ -0,0 +1,7 @@
+---
+'@lit-labs/ssr-dom-shim': minor
+'@lit-labs/ssr': minor
+'@lit-labs/ssr-react': patch
+---
+
+Implement SSR event handling
```

Comment:
Can you mention the option that enables this here?

Reply:
You mean like this?
```suggestion
Implement SSR event handling and an optional flag `globalThis.litSsrCallConnectedCallback` to call `connectedCallback` during SSR, if set to true.
```